src/plugins/drawing/objects/drawing-object.ts

```diff
@@ -84,6 +89,17 @@ export const DrawingObject = types.model("DrawingObject", {
     self.x = self.dragX ?? self.x;
     self.y = self.dragY ?? self.y;
     self.dragX = self.dragY = undefined;
+  },
+  dragBounds(deltas: BoundingBoxDelta) {
```

Comment:
I'd probably call this `setDragBounds`. `dragBounds` makes it sound like a getter.